app/views/account_deletions/edit_reason.html.slim

```diff
@@ -0,0 +1,18 @@
+.govuk-grid-row
+  .govuk-grid-column-two-thirds-from-desktop
+    = govuk_back_link(href: user_path)
+
+    = form_with model: @user, url: update_reason_user_account_deletion_path  do |f|
+      = f.govuk_error_summary
+      = f.govuk_radio_buttons_fieldset(:deleted_reason,
+        legend: { text: 'Tell us why you want to close your account' },
+        hint: { text: 'Knowing why you want to close your account will help us improve the training for everyone.' }) do
+        = f.govuk_radio_button :deleted_reason, 'not useful', label: { text: 'I did not find the training useful' }, link_errors: true
+        = f.govuk_radio_button :deleted_reason, 'no time', label: { text: 'I did not have time to do the training' }
+        = f.govuk_radio_button :deleted_reason, 'no longer in early years', label: { text: 'I no longer work in early years' }
+        = f.govuk_radio_button :deleted_reason, 'too many emails', label: { text: 'You sent me too many emails' }
+        = f.govuk_radio_button :deleted_reason, 'other', label: { text: 'Another reason' } do
```

Comment:
Can you try changing the value on this to something other than `other` - I think that caused my previous problem....

Reply:
That doesnt seem to have helped, sadly